### ISSUE TITLE
Filtrer entrées inutiles

### ISSUE THREAD

**Issue by RenaudLou** [pgrimaud/lametric-ratp#5]
Bonjour
Serait-il possible de filtrer les prochains passages non pertinents ? Par exemple sur le RER :
- train sans arrêt
- train à quai (trop tard pour l'avoir)
- train à l'approche (si on a mis délai > 0)

Et merci pour cette appli très pratique !

**Comment by pgrimaud** [comment]
Hello, 

Yes tout à fait.
Tu aurais le listing des termes de ce que tu veux retirer ?

**Comment by RenaudLou** [comment]
Oui, de ce que je vois par échantillonnage il semble y avoir 2 formulations selon que le numéro du quai est affiché (Laplace, Bourg La Reine...) ou pas :
Train sans arrêt
Sans arrêt V.1 
Train à quai
Train à quai V.1
Train à l'approche
A l'approche V.2B
Donc pour les termes à retirer avec regexp ça serait :
Train sans arrêt
Sans arrêt.*
Train à quai.*
Train à l'approche
A l'approche.*
